docs/API.md

```diff
@@ -126,6 +127,23 @@ for await(const item of filtered) {
 }
 ```
 
+## first
+
+Take the first value of ansync iterable that fullfills the predicate. If not predicate is provided, it returns the first value of the async iterable.
+
+```javascript
+import { first } from "axax/es5/first";
+import { of } from "axax/es5/of";
+
+const firsted = filter(
```

Comment:
this should be `first` instead

Reply:
Not sure that would make sense, we already imported the variable `first` on line 135.

Reply:
Oh, I just mean that it's using `filter` but I think it should be `first` instead

Reply:
Oh, of course! Now I see what you mean. Fixing!